test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup.yml

```diff
@@ -0,0 +1,4 @@
+- hosts: localhost
+  roles:
+    - role: aws_ssm_integration_test_setup
+      when: ansible_python_version is version('3', '<')
```

Comment:
Why is this test being limited to Python 2?

Reply:
@mattclay ,
AWS SSM Connection plugin currently not supporting Python 3. So we set the integration test execution limited to Python 2.

Reply:
Support (and tests) for Python 3 need to be implemented before this can be merged.

Reply:
https://github.com/ansible/ansible/pull/49652/commits/2c2a5b70573bcf61ea83476b7bcd2aca2fda19dc adds support for python3